tools/grab_micro_posts.py

```
@@ -142,18 +166,18 @@ def main():
     new_posts_created = 0
     for entry in sorted_entries:
         new_note_id = highest_note_id + new_posts_created + 1
-        print(f"Attempting to create post with Note ID: {new_note_id}")
+        logging.info(f"Attempting to create post with Note ID: {new_note_id}")
         if create_hugo_content(entry, hugo_content_dir, new_note_id):
             new_posts_created += 1
 
-    print(f"Processed {len(sorted_entries)} entries.")
-    print(f"Created {new_posts_created} new posts.")
-    print(f"New highest Note ID: {highest_note_id + new_posts_created}")
+    logging.info(f"Processed {len(sorted_entries)} entries.")
```

Ruff (0.8.2)

Logging statement uses f-string

(G004)